docs/Configuration.md

```diff
@@ -1,40 +1,213 @@
+
+[syslogreceiver_readme]: https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/v0.27.0/receiver/syslogreceiver
+
+### Statsd Receiver
+
+StatsD Receiver ingests [StatsD messages][statsd_messages] into the OpenTelemetry Collector.
 
-Link to [Syslog Receiver detailed configuration][syslogreceiver_readme]
+The basic configuration for StatsD Receiver has following format:
+
+```yaml
+receivers:
+  statsd:
+  statsd/2:
+    endpoint: "localhost:8127"
+    aggregation_interval: 70s
+    enable_metric_type: true
+    timer_histogram_mapping:
+      - statsd_type: "histogram"
+        observer_type: "gauge"
+      - statsd_type: "timing"
+        observer_type: "gauge"
+```
+
+for details please see [StatsD Receiver documentation][statsdreceiver_readme].
+
+[statsd_messages]: https://github.com/statsd/statsd/blob/master/docs/metric_types.md
+[statsdreceiver_readme]: https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/release/v0.27.x/receiver/statsdreceiver
 
 ### Telegraf Receiver
 
-Telegraf Receiver simple config
+Telegraf Receiver ingests metrics from various [input plugins][input_plugins]
+into OTC pipeline.
+
+The basic configuration for Telegraf Receiver has following format:
+
+```yaml
+receivers:
+  telegraf:
+    separate_field: false
+    agent_config: |
+      [agent]
+        interval = "3s"
+        flush_interval = "3s"
+      [[inputs.mem]]
+```
 
-Link to [Telegraf Receiver detailed configuration][telegrafreceiver_readme]
+for details please see [Telegraf Receiver documentation][telegrafreceiver_readme].
+
+[input_plugins]: https://github.com/influxdata/telegraf/tree/master/plugins/inputs
+[telegrafreceiver_readme]: ../pkg/receiver/telegrafreceiver
 
 ### OTLP Receiver
 
-OTLP Receiver simple config
+OTLP Receiver receives data via gRPC or HTTP using [OTLP][otlp] format.
 
-Link to [OTLP Receiver detailed configuration][otlpreceiver_readme]
+The basic configuration for OTLP Receiver has following format:
 
-## Processors
+```yaml
+receivers:
+  otlp:
+    protocols:
+      grpc:
+      http:
+```
 
-## Exporters
+for details please see [OTLP Receiver documentation][otlpreceiver_readme]
 
-[filelogreceiver_readme]: https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/v0.27.0/receiver/filelogreceiver
-[syslogreceiver_readme]: https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/v0.27.0/receiver/syslogreceiver
+[otlp]: https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/protocol/otlp.md
 [otlpreceiver_readme]: https://github.com/open-telemetry/opentelemetry-collector/tree/v0.27.0/receiver/otlpreceiver
-[telegrafreceiver_readme]: ../pkg/receiver/telegrafreceiver
\ No newline at end of file
+
+### Receivers from OpenTelemetry Collector
+
+Sumo Logic OT Distro has inbuilt receivers from [OpenTelemetry Collector](https://github.com/SumoLogic/opentelemetry-collector) and they can be used in configuration for this distribution.
```

Comment:
```suggestion
Sumo Logic OT Distro has builtin receivers from 
[OpenTelemetry Collector](https://github.com/SumoLogic/opentelemetry-collector) and they can be used
in configuration for this distribution.
```

Or perhaps let's move the link out of the paragraph and then restructure this, hm?